marketplace_docs/templates/Fabric/interactive_script.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+clear
+
+cat <<EOM
+
+Zulip will ask you for the following two input for completing the installation:
+
+- The email address of the person or team who should get support and error emails from this Zulip server.
+- The user-accessible domain name for this Zulip server, i.e., what users will type in their web browser.
+
+To skip the Zulip setup for now, press Ctrl+C.  You will be prompted again on your next login.
+
+You can also re-run this setup script at any time with the command:
+   bash $(readlink -f "${0}")
+
+Please press enter when you are ready to configure Zulip.
+
```

Comment:
These text snippets are tough -- I haven't installed enough software recently to have a good sense of what's normal. But one pass:

```
Welcome! Zulip will ask for the following two pieces of information during setup:
- An email address for support and error emails, like ops@example.com
- A hostname for the Zulip server, like chat.example.com

To skip Zulip setup for now, press Ctrl+C.  You will be prompted again on your next login.

You can also re-run this setup script at any time with the command
   bash $(readlink -f "${0}")

Press Enter to continue setup.
```

Reply:
This looks totally reasonable.  I would put this somewhere in there (maybe just before the "To skim ..." line.

"For more information on these options, check out:
     https://zulip.readthedocs.io/en/latest/production/install.html#installer-options
"

Reply:
they won't get the 2 cert-related options though, right? Just email and hostname?

Reply:
This installer just uses certbot by default.  

Which maybe means these docs should tell them to first register the hostname in DNS or otherwise mention that?

Reply:
yeah, that seems reasonable.

Also I'd be worried about linking to a doc with 4 installer options, and then the user feeling like something went wrong when they only see 2 of the 4 during setup?

Maybe we can add a line to that rtd article to clarify (e.g. "Automatic installers only give the first two options.")